.buildkite/pull-requests.json

```diff
@@ -13,7 +13,12 @@
         "always_trigger_comment_regex": "^(?:(?:buildkite\\W+)?(?:build|test)\\W+(?:this|it))|^/test$",
         "skip_ci_labels": [ ],
         "skip_target_branches": [ ],
-        "skip_ci_on_only_changed": [ ],
+        "skip_ci_on_only_changed": [
+          "^docs/",
+          "^.github/",
+          "^.mergify.yml",
+          "\\.md$"
```

Comment:
`README.md` in the root of the directory is generated in the CI steps.

Probably it needs to be removed from here in case someone updates it but not modifies the corresponding template:
https://github.com/elastic/elastic-package/tree/main/tools/readme

Reply:
About the others I don't recall now any usage of them in the Buildkite builds.

Shuold we remove any other regex @jsoriano?

Reply:
see [`b0a5095` (#2368)](https://github.com/elastic/elastic-package/pull/2368/commits/b0a5095e11d6bcef5aa825ae77000146f84e0997)